source/renderer/app/components/voting/VotingUnavailable.js

```diff
@@ -1,132 +1,30 @@
-    description: 'First link text on the "Voting unavailable" screen',
-  },
-  link2Text: {
-    id: 'voting.unavailable.link2Text',
-    defaultMessage: '!!!Project Catalyst Chat',
-    description: 'Second link text on the "Voting unavailable" screen',
-  },
-  link1Url: {
-    id: 'voting.unavailable.link1Url',
-    defaultMessage: '!!!https://t.me/cardanocatalyst',
-    description: 'First link URL on the "Voting unavailable" screen',
-  },
-  link2Url: {
-    id: 'voting.unavailable.link2Url',
-    defaultMessage: '!!!https://t.me/ProjectCatalystChat',
-    description: 'Second link URL on the "Voting unavailable" screen',
-  },
-});
-
 type Props = {
   syncPercentage: number,
-  isVotingRegistrationAvailable: boolean,
-  onExternalLinkClick: Function,
 };
 
-@observer
-export default class VotingUnavailable extends Component<Props> {
-  static contextTypes = {
-    intl: intlShape.isRequired,
-  };
-
-  render() {
-    const { intl } = this.context;
-    const {
-      syncPercentage,
-      isVotingRegistrationAvailable,
-      onExternalLinkClick,
-    } = this.props;
-
-    const heading = intl.formatMessage(messages.heading);
-    const paragraph1 = intl.formatMessage(messages.paragraph1, {
-      currentFundNumber: CURRENT_VOTING_FUND_NUMBER,
-      nextVotingFundNumber: NEXT_VOTING_FUND_NUMBER,
-    });
-    const link1 = (
-      <Link
-        className={styles.link}
-        label={intl.formatMessage(messages.link1Text)}
-        onClick={() =>
-          onExternalLinkClick(intl.formatMessage(messages.link1Url))
-        }
-      />
-    );
-    const link2 = (
-      <Link
-        className={styles.link}
-        label={intl.formatMessage(messages.link2Text)}
-        onClick={() =>
-          onExternalLinkClick(intl.formatMessage(messages.link2Url))
-        }
-      />
-    );
-
-    return (
-      <div className={styles.component}>
-        {isVotingRegistrationAvailable ? (
-          <>
-            <LoadingSpinner big />
-            <div className={styles.description}>
-              <FormattedHTMLMessage
-                {...globalMessages.featureUnavailableWhileSyncing}
-                values={{
-                  syncPercentage: new BigNumber(syncPercentage).toFormat(2),
-                }}
-              />
-            </div>
-          </>
-        ) : (
-          <>
-            <h1>{heading}</h1>
-            <p>{paragraph1}</p>
-            <FormattedMessage
-              {...messages.paragraph2}
-              values={{ link1, link2 }}
-              tagName="p"
-            />
-          </>
-        )}
+const VotingUnavailable = ({ syncPercentage }: Props) => {
+  return (
+    <div className={styles.component}>
+      <LoadingSpinner big />
+      <div className={styles.description}>
+        <FormattedHTMLMessage
+          {...globalMessages.featureUnavailableWhileSyncing}
+          values={{
+            syncPercentage: new BigNumber(syncPercentage).toFormat(2),
```

Comment:
Maybe you could use `formattedNumber` from formatters.js

Reply:
Updated. Thanks!